src/main/resources/db/migration/V23__flatten_categories_and_audit_benefit_structuring.sql

```
@@ -0,0 +1,337 @@
+    ('CONVENIENCE_STORE', 'CU', TRUE),
+    ('CONVENIENCE_STORE', '세븐일레븐', TRUE),
+    ('CONVENIENCE_STORE', '이마트24', TRUE),
+    ('RESTAURANT', '아웃백', TRUE),
+    ('RESTAURANT', 'VIPS', TRUE),
+    ('ONLINE_SHOPPING', '농협몰', FALSE);
+
+INSERT INTO merchants
+    (merchant_id, merchant_category_id, name, normalized_name, status,
+     has_physical_location, created_at, updated_at)
+SELECT UUID(), category.merchant_category_id,
+       seed.merchant_name, seed.merchant_name, 'active', seed.has_physical_location,
```

SQLFluff (4.2.2)

[error] Reuse of column alias 'merchant_name' from line 114.

(AL08)